src/v2/Apps/Artist/Routes/AuctionResults/ArtistAuctionResultItem.tsx

```diff
@@ -388,26 +388,17 @@ const renderPricing = (
   }
 }
 
-const renderEstimate = (estimatedPrice, user, mediator, size) => {
-  const justifyContent = size === "xs" ? "flex-start" : "flex-end"
```

Comment:
When solving the alignment inconsistencies I learned that everything should just be aligned `flex-start`.